prowler/providers/azure/services/containerregistry/containerregistry_service.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+
+from azure.mgmt.containerregistry import ContainerRegistryManagementClient
+
+from prowler.lib.logger import logger
+from prowler.providers.azure.azure_provider import AzureProvider
+from prowler.providers.azure.lib.service.service import AzureService
+from prowler.providers.azure.services.monitor.monitor_client import monitor_client
+from prowler.providers.azure.services.monitor.monitor_service import DiagnosticSetting
+
+
+########################## Container Registry
+class ContainerRegistry(AzureService):
+    def __init__(self, provider: AzureProvider):
+        super().__init__(ContainerRegistryManagementClient, provider)
+        self.registries = self._get_container_registries(provider)
+
+    def _get_container_registries(self, provider):
```

Comment:
see above.

Reply:
It is mandatory in the father constructor, in the methods that you define is not mandatory because you are not using in any line of the method. The above case was a fail, I only wanted to change this one.

Reply:
Ok understood. Is fixed.